avro-serializer/src/main/java/io/confluent/kafka/serializers/KafkaAvroDecoder.java

```diff
@@ -30,7 +30,7 @@ public KafkaAvroDecoder(SchemaRegistryClient schemaRegistry) {
 
   public KafkaAvroDecoder(SchemaRegistryClient schemaRegistry, VerifiableProperties props) {
```

Comment:
Why do we still keep the VerifiableProperties here?

Reply:
There's a comment saying that the Kafka consumer relies on the `KafkaAvroDecoder` constructor that takes a `VerifiableProperties`.